Carthage/Checkouts/xcconfigs

```diff
@@ -1 +1 @@
-Subproject commit d78854b22b9567e7f30d748bdd4966c86bcc93a5
+Subproject commit ec5753493605deed7358dec5f9260f503d3ed650
```

Comment:
Unfortunately, you mistakenly changed the submodules when you merged `master` in. That's always really confusing when you get a merge conflict. 😩

Running `carthage checkout --use-submodules` should fix it up.